sympy/core/tests/test_evaluate.py

```diff
@@ -44,22 +45,23 @@ def test_add():
         assert S(2) / 4 * 4 == Mul(S(2) / 4, 4)
         assert 4 * (S(2) / 4) == Mul(4, S(2) / 4)
 
-        assert S(6) / 3 == Mul(6, S(1) / 3)
+        assert S(6) / 3 == Mul(6, Pow(3, -1))
         assert S(1) / 3 * 6 == Mul(S(1) / 3, 6)
 
         assert S(1) / 3 + sqrt(3) == Add(S(1) / 3, sqrt(3))
         assert sqrt(3) + S(1) / 3 == Add(sqrt(3), S(1) / 3)
 
         assert S(1) / 2 * 10.333 == Mul(S(1) / 2, 10.333)
-        assert 10.333 * S(1) / 2 == Mul(10.333, S(1) / 2)
+        assert 10.333 * (S(1) / 2) == Mul(10.333, (S(1) / 2))
```

Comment:
or `assert 10.333 * (S(1) / 2) == Mul(10.333, 2**-1)` ? or simply no parens on the right. just a nit.